src/components/link/link.spec.tsx

```diff
@@ -293,6 +293,18 @@ describe("Link", () => {
         expect(anchor.getAttribute("aria-label")).toEqual("test");
       });
     });
+
+    describe("when removeAriaLabelOnIcon is true", () => {
+      it("should set aria-label as undefined on icon", () => {
+        wrapper = renderLink({
```

Comment:
**suggestion** (non-blocking): I would feel this test was much more robust if we also passed an `ariaLabel` prop to the Link, to ensure that the `removeAriaLabelOnIcon` is really doing its job

Reply:
Not an unreasonable suggestion and makes sense from a robustness perspective. I'll get it done now and pushed back up 👍🏻